src/v1/key_management_service_client.ts

```diff
@@ -3415,23 +3416,23 @@ export class KeyManagementServiceClient {
    */
 
   getIamPolicy(
```

Comment:
We need to add jsdoc comments for these methods.

Reply:
sounds great to me, let me check is there anydoc I can import for this.

Reply:
stole it from original nodejs-kms repo. Thanks!